client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -514,6 +523,51 @@ class CancelPurchaseForm extends Component {
 				</div>
 			);
 		}
+
+		if ( surveyStep === REMOVE_PLAN_STEP ) {
+			return (
+				<div className="cancel-purchase-form__remove-plan">
+					<FormattedHeader
+						brandFont
+						headerText={ translate( 'Sorry to see you go' ) }
+						subHeaderText={
+							<>
+								<span className="cancel-purchase-form__remove-plan-text">
+									{
+										// Translators: %(planName)s: name of the plan being canceled, eg: "WordPress.com Business"
+										translate(
+											'If you remove your subscription, you will lose access to the features of the %(planName)s plan.',
```

Comment:
This only valid if `const isPlanPurchase = isPlan( purchase );` is true, which we don't check here.

Reply:
@mmtr this condition is only met when `skipSurvey` is `true`, which is set on L679 of `client/me/purchases/manage-purchase/index.tsx`. I could rename the prop to `skipRemovePlanSurvey`, so its intent is clearer. Wdyt?

Reply:
> this condition is only met when skipSurvey is true, which is set on L679 of client/me/purchases/manage-purchase/index.tsx.

As far as I can see in that line, it can be either a plan or something else:

https://github.com/Automattic/wp-calypso/blob/002968d0b87edb1bffaa64b344913c4fd7c73934/client/me/purchases/manage-purchase/index.tsx#L659-L665

So `skipSurvey=true` doesn't imply that we're removing a plan. Therefore, the content of the "Sorry to see you go" needs to be adapted.

Reply:
@mmtr right, I've addressed it on https://github.com/Automattic/wp-calypso/pull/94901/commits/7c0ef68a9d1b3d4ee6214dddeceb69f4606f1dda